lib/processors/jsdoc/jsdocGenerator.js

```diff
@@ -93,7 +89,7 @@ async function generateJsdocConfig({targetPath, tmpPath, namespace, projectName,
 	const backslashRegex = /\\/g;
 
 	// Resolve path to this script to get the path to the JSDoc extensions folder
-	const jsdocPath = path.normalize(__dirname);
+	const jsdocPath = path.normalize(import.meta.dirname);
```

Comment:
This one seems to be added in Node 21.2.1 https://nodejs.org/api/esm.html#importmetadirname and there's a chance we might fail in certain cases

Reply:
See also the documentation pages:
v20: https://nodejs.org/docs/latest-v20.x/api/esm.html#importmetadirname
v21: https://nodejs.org/api/esm.html#importmetadirname

Reply:
Thanks!